lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -108,7 +108,7 @@ func resetDBWriteConection(dbFlag DbType) {
 			config := getInMemoryDBConfig()
 			log.Debug("getInMemoryDBConfig: ", config)
 			currentMasterIP, currentMasterPort := GetCurrentMasterHostPort(config)
-			log.Info("InMemory currentMaster IP: "+currentMasterIP, "CurrentMaster Port: "+currentMasterPort, "inMemDBConnPool.MasterIP: "+inMemDBConnPool.MasterIP)
+			log.Info("InMemory currentMaster IP: "+currentMasterIP, ", CurrentMaster Port: "+currentMasterPort, " ,InMemDBConnPool.MasterIP: "+inMemDBConnPool.MasterIP)
```

Comment:
`+currentMasterIP,`  shouldn't be `+currentMasterIP+`
same with `+currentMasterPort,`

Reply:
addressed.